packages/core-mobile/app/navigation/NavUtils.tsx

```diff
@@ -106,8 +107,15 @@ export const TabButton = ({
   image,
   focused
 }: TabButtonProps): JSX.Element => {
+  const { bottom } = useSafeAreaInsets()
```

Comment:
let's use the useVariableSafeAreaInsets hook everywhere to avoid confusion since `bottom` or `top` is the same between these two